api/agentpb/agent.proto

```diff
@@ -329,6 +329,73 @@ message CheckConnectionResponse {
   Stats stats = 2;
 }
 
+// JobStatusRequest is a ServerMessage asking pmm-agent for job status.
+message JobStatusRequest {
```

Comment:
Do we use it somewhere?

Reply:
Managed doesn't request it anywhere now, but it allows to check if the job alive or dead.